### PR TITLE
Change usb enumeration to only specific Vid/Pid

### PR DESCRIPTION
**Pull request by sctanf**
fix hidtracker messing with random usb devices like mice and whatever else

### CODE REVIEW

**Comment by gorbit99** [comment]
I checked, it seems to solve the issue for me, and dongles still seem to work.

File: server/desktop/src/main/java/dev/slimevr/desktop/tracking/trackers/hid/TrackersHID.kt [hunk 1]
```diff
@@ -227,25 +248,51 @@ class TrackersHID(name: String, private val trackersConsumer: Consumer<Tracker>)
 		}
 	}
 
+	private fun deviceEnumerate() {
+		var root: HidDeviceInfoStructure? = null
+		try {
+			root = HidApi.enumerateDevices(0x2FE3, 0x5652) // TODO: change to proper vendorId and productId, need to enum all appropriate productId
+		} catch (e: Throwable) {
+			// i dont know enough kotlin for this
```
Comment (ImUrX) [inline server/desktop/src/main/java/dev/slimevr/desktop/tracking/trackers/hid/TrackersHID.kt:256]:
```suggestion
			LogManager.severe("[TrackerServer] Couldn't enumerate HID devices", e)
```

File: server/desktop/src/main/java/dev/slimevr/desktop/tracking/trackers/hid/TrackersHID.kt [hunk 2]
```diff
@@ -227,25 +248,51 @@ class TrackersHID(name: String, private val trackersConsumer: Consumer<Tracker>)
...
+	}
+
 	override fun run() { // Doesn't seem to run
 	}
 
 	fun getDevices(): List<Device> = devices
 
-	override fun hidDeviceAttached(event: HidServicesEvent) {
-		checkConfigureDevice(event.hidDevice)
+	override fun hidDeviceAttached(event: HidServicesEvent) { // We dont use these
```
Comment (ImUrX) [inline server/desktop/src/main/java/dev/slimevr/desktop/tracking/trackers/hid/TrackersHID.kt:286]:
```suggestion
	// We don't use these
	override fun hidDeviceAttached(event: HidServicesEvent) {
```

**Review by ImUrX — CHANGES_REQUESTED** [review]

File: server/desktop/src/main/java/dev/slimevr/desktop/tracking/trackers/hid/TrackersHID.kt [hunk 3]
```diff
@@ -227,25 +248,51 @@ class TrackersHID(name: String, private val trackersConsumer: Consumer<Tracker>)
 		}
 	}
 
+	private fun deviceEnumerate() {
+		var root: HidDeviceInfoStructure? = null
+		try {
+			root = HidApi.enumerateDevices(0x2FE3, 0x5652) // TODO: change to proper vendorId and productId, need to enum all appropriate productId
```
[the remark this answers is not in the document]
Comment (sctanf) [reply server/desktop/src/main/java/dev/slimevr/desktop/tracking/trackers/hid/TrackersHID.kt:254]: I assume later it will be replaced with a list?

**Review by ImUrX — APPROVED** [review]

**Review by Eirenliel — APPROVED** [review]

**Pull request merged**